tools/gfx-unit-test/cache-eviction-policy-expected-d3d12-1.txt

```diff
@@ -0,0 +1,2 @@
+DA799C6427056E9E291844A469A7049D 5C5B27A848BE6D2F974B65DFD3E548E9
```

Comment:
Can we just define this directly as constants in test code? That way we don't need these separate files.

Reply:
Yeah I can do this the same way as the shader code since Git's auto-convert of LF to CRLF is likely breaking the `memcmp()` right now.

Reply:
The problem with this kind of baked in digest is that different downstream compiler versions on different systems will produce different digest. Instead of hard coding, we need a better way to tell if the eviction policy is correct that adapts to different digests.

Reply:
Unrelated change, but I've removed all the shader modifications. Other tests in this file already do plenty of shader modifying, and the central focus of this test is to confirm that the entries are being added in the correct order and are being removed correctly when the cache reaches size limits. The shader modifications make this more complex than it needs to be.

Instead of the hard-coded values, I've changed the test to save the contents of the cache file after each step. Since no shaders are being modified, a shader will correspond to the same line in all versions of the file, which allows us to check that the files themselves are in the right orders by checking that certain positions in the lists are equal to each other. Maybe not as straightforward as fetching the hashes first before anything else and doing straight comparisons, but that idea sounded a bit terrible in my head.